.gitignore

```diff
@@ -7,3 +7,10 @@ npm-debug.log
 secrets.js
 /webpack.config.js
 /dist/bundle.js
+
+log.txt
+pouch__all_dbs__
+crops
+_users
+_replicator
+/docs/generated/*
```

Comment:
`docs/generated/` are meant to be committed as they are exported to https://ecohackerfarm.github.io/powerplant/, so this line should be removed?